nuxeo-services/nuxeo-platform-convert/src/main/java/org/nuxeo/ecm/platform/convert/plugins/JODBasedConverter.java

```diff
@@ -183,8 +183,18 @@ public BlobHolder convert(BlobHolder blobHolder, Map<String, Serializable> param
         // This plugin do deal only with one input source.
         String sourceMimetype = inputBlob.getMimeType();
 
-        boolean pdfa1 = parameters != null && Boolean.TRUE.equals(parameters.get(PDFA1_PARAM));
-
+        boolean pdfa1 = false;
+        
+        if (parameters != null) {
+        	Serializable pdfa1Val = parameters.get(PDFA1_PARAM);
+        	
+	        if (pdfa1Val instanceof Boolean) {
+	        	pdfa1 = (Boolean) pdfa1Val;
```

Comment:
`pdfa1 = ((Boolean) pdfa1Val).booleanValue()` to avoid auto-unboxing (which is a warning in some IDEs).